### PR TITLE
Add Ukrainian translation

### PR DESCRIPTION
**Pull request by AlexHladin**
I've added the localization file for the Ukrainian language and registered it in the app_language_changer.dart file. Things that could be translated are translated.

### CODE REVIEW

File: lib/localization/app_uk.arb [hunk 1]
```diff
@@ -0,0 +1,1060 @@
+{
+  "appTitle": "Paisa",
+  "@appTitle": {
+    "description": "Назва додатку",
+    "type": "text"
+  },
```
Comment (SahSantoshh) [inline lib/localization/app_uk.arb:6]: @AlexHladin we only need key and value to be translated. Could you please update this by remove all key with `@` :)

**Review by SahSantoshh — CHANGES_REQUESTED** [review]

File: lib/localization/app_uk.arb (hunk 1 above)
Comment (SahSantoshh) [reply lib/localization/app_uk.arb:6]: for more info, please check other trnaslation files/

Comment (SahSantoshh) [reply lib/localization/app_uk.arb:6]: For more info, please check other translation files

Comment (AlexHladin) [reply lib/localization/app_uk.arb]: @SahSantoshh updated :)

**Review by SahSantoshh — APPROVED** [review]

**Comment by h4h13** [comment]
This for merged in #447 closing

**Pull request closed without merging**